webpack.prod.config.babel.js

```diff
@@ -30,9 +30,9 @@ export default {
     filename: '[name]-[contenthash].js',
```

Comment:
On L18 (I cannot comment on it), there is a `STATIC_DIR` variable that should be updated with `config.get('staticPath').replace(/\//g, '')` instead of `static` (we cannot use `replaceAll()` yet)

Reply:
You can try the config locally by:

1. rename the `local.js.dist` file to `local.js`
2. execute `NODE_CONFIG_ENV=dev NODE_ENV=production yarn build`
3. execute `NODE_CONFIG_ENV=dev NODE_ENV=production yarn start`

Reply:
mmm, I didn't want to change the target directory, since ops scripts depend on the statics being in `dist/static/` at the moment (which is nice cause it's entirely backwards-compatible).

Reply:
In this case, we might want to:

1. update the assets middleware to use the `staticPath` config param
2. use `static-frontend` for all envs

I'd prefer (2) but up to you.